src/main/cli/cli.c

```diff
@@ -679,9 +679,11 @@ static void backupPgConfig(const pgRegistry_t *pg)
     memcpy(pg->copy, pg->address, pg->size);
 }
 
-static void restorePgConfig(const pgRegistry_t *pg)
+static void restorePgConfig(const pgRegistry_t *pg, uint16_t notToRestoreGroupId)
```

Comment:
i did not quite understand why we need "notToRestoreGroupId"
Also, what is "bare" option in CLI?
Thanks!

Reply:
Well, essentially this is a negative mask - the way that resetting only a single parameter group works is that we first create a backup of all current settings, then we reset the entire config to defaults (we have to do this as there is no way to apply only a subset of the custom defaults in a targeted way), and then we restore the previously backed up settings _except for the one parameter group we want to reset_. :grin: 
The `bare` option is used to generate a `diff` / `dump` that is bare commands, without all of the extra things like commands to report errors.